content/Mobile/Android/index.adoc

```diff
@@ -1,29 +1,46 @@
 == Getting Started with Android
 
-Native support for WebAuthn and FIDO is finally here for Android. Yubico provides a comprehensive library for developers to interact with YubiKeys from their Android applications. 
+Our new Android SDK, the YubiKit for Android, provides native support for WebAuthn. Yubico offers two great user experiences on Android employing a simple tap for NFC or a physical connection with USB.
```

Comment:
WebAuthn should be removed from this sentence, since the SDK does not provide it. The NOTE further down already references the official Google APIs for that usecase.

Reply:
If we displayed the content of the README.md from the android-yubikit repo here, then we would be rid of that marketing-flavored section altogether.

Reply:
correction made